wsdl/ver10/schema/metadatastream.xsd

```diff
@@ -102,6 +102,11 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 			<xs:element name="Extension" type="tt:ShapeDescriptorExtension" minOccurs="0"/>
 			<xs:any namespace="##any" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>	<!-- reserved for ONVIF -->
 		</xs:sequence>
+		<xs:attribute name="Distance" type="xs:float">
+			<xs:annotation>
+				<xs:documentation>Distance in meters to the object.</xs:documentation>
```

Comment:
does this parameter can be made as optional?

Reply:
Attributes are optional by default

Reply:
thanks fredrik for clarification.